quit/tools/evaluate.py

```diff
@@ -480,8 +480,10 @@ def evalQuery(graph, query, initBindings, base=None):
 
                 ctx.load(d.default, default=True)
 
+            # TODO re-enable original behaviour if fixed
```

Comment:
How will this be fixed? Please add a n issue for this if this is really necessary else remove the out commented lines.